export.py

```diff
@@ -624,6 +624,7 @@ def run(data=ROOT / 'data/coco128.yaml',  # 'dataset.yaml path'
     assert sum(flags) == len(include), f'ERROR: Invalid --include {include}, valid --include arguments are {formats}'
     jit, onnx, xml, engine, coreml, saved_model, pb, tflite, edgetpu, tfjs = flags  # export booleans
     file = Path(url2file(weights) if str(weights).startswith(('http:/', 'https:/')) else weights)  # PyTorch weights
+    data = check_file(data)
```

Comment:
@KSGulin `data` is originally a `Pathlib.PosixPath` object, but `check_file` might return a string, would it make sense to explicitly wrap data in a `Path` object?
```suggestion
    data = Path(check_file(data))
```

Reply:
Path is the default value for data, but when user provided it takes on a str value. So it's meant to be compatible with both!